src/components/Authorization/Authorization.css

```diff
@@ -87,34 +88,32 @@ a{
 }
 
 .eye-icon {
+  padding:10px;
   position: absolute;
-  top: calc(36.6% - 8px);
-  left: calc(75% - 8px)
+  top: calc(41% - 8px);
+  left: calc(88% - 8px)
```

Comment:
What about little explain, what happens here? Why we have these magic numbers like 41/88%? 
Another question for this item: why this item placed outside at input? We should place it inside the input/label and use position relative for parent and position absolute for this element. Also, if we have something like icon - we should create this like icon - with svg.

Reply:
fix